### ISSUE TITLE
Service does not work when the project's folder contains a non-ASCII character

### ISSUE THREAD

**Issue by molnar-a** [angular/vscode-ng-language-service#292]
My project folder's name contained an 'á' character, and the language service did not work. In the language service output terminal the 'Search path' string was displayed wrong. After moving the project to a folder containing only ASCII characters, the service worked properly.

**Comment by 1002237913** [comment]
+1

**Comment by alkthegit** [comment]
I'm having the same issue.
When after installing in VS Code Angular Language Service didn't work as expected in a template html file, it's output was like this:

```bash
Search path: d:/Projects/Ð°Ð½Ð³ÑÐ»ÑÑ/src/app
No config files found.
...
```

The encoding here in the output is ISO-8859-1 (Latin-1).

So my first guess was to rename the folder so that it contains only characters from that coding page and it started working.
With no coloring, but at least with syntax check and so on.

Clearly needs to be fixed.

**Comment by lightman0215** [comment]
+1

**Comment by kyliau** [comment]
This is fixed in v0.900.0, please download from the Releases tab in this repo to try it out.
It'll be released to the vscode marketplace in a few weeks.
